src/pages/debug.tsx

```diff
@@ -34,6 +35,7 @@ export default function DebugPage() {
       <EnvStatus className="DebugPage__Section" />
       <BudgetsUpdate className="DebugPage__Section" />
       <ErrorReporting className="DebugPage__Section" />
+      <RemoveVerification className="DebugPage__Section" />
```

Comment:
This should also be remove before prod right?

Reply:
Yep!